music/src/piano_genie/model.ts

```diff
@@ -291,6 +361,75 @@ class PianoGenie {
     disposeState(this.lastState);
     this.initialized = false;
   }
+}
+
+/**
+ * Original Piano Genie config with autoregression and delta time features.
+ */
+class PianoGenieAutoDt extends PianoGenieBase {
```

Comment:
Since this is getting aliased anyway, can it maybe use the full name to be more clear? `PianoGenieAutoregressionDeltaTime`?

Reply:
Yes!